gammapy/spectrum/spectrum_fit.py

```diff
@@ -210,15 +199,17 @@ def _run_sherpa_fit(self):
         """Plain sherpa fit using the session object
         """
         from sherpa.astro import datastack
-        log.info("Starting SHERPA")
-        log.info(str(self))
+        from sherpa.utils.err import IdentifierErr
+        
+        # TODO: Make this optional
```

Comment:
> You say I should not mute the logger from here, right?

Yes.

> The problem is that, since sherpa is imported in this function (and typically not in a user script) I only have access to the logger from here. I muted it because it is annoying when fitting O(100) simulated spectra. Do you propose to leave the logger as is and make users 'hack' it away (by importing sherpa) themselves?

Yes, why not let users configure the Sherpa logger from their script?

If it helps, you can expose a convenience option for that from `gammapy.spectrum`.
But I think all that's needed here is some examples or two sentences in the docs to tell people how to configure the Sherpa logger?